pkg/operator/controllers/routefix/routefix.go

```diff
@@ -11,25 +11,63 @@ import (
 	securityv1 "github.com/openshift/api/security/v1"
 	appsv1 "k8s.io/api/apps/v1"
 	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/kubernetes/pkg/apis/rbac"
 
 	arov1alpha1 "github.com/Azure/ARO-RP/pkg/operator/apis/aro.openshift.io/v1alpha1"
 	"github.com/Azure/ARO-RP/pkg/util/version"
 )
 
 const (
-	kubeName           = "routefix"
-	kubeNamespace      = "openshift-azure-routefix"
-	kubeServiceAccount = "system:serviceaccount:" + kubeNamespace + ":default"
-	shellScript        = `for ((;;))
+	kubeName            = "routefix"
+	kubeNamespace       = "openshift-azure-routefix"
+	kubeServiceAccount  = "system:serviceaccount:" + kubeNamespace + ":default"
+	containerNameDrop   = "drop-icmp"
+	containerNameDetect = "detect"
+	shellScriptLog      = `while true;
 do
-  if ip route show cache | grep -q 'mtu 1450'; then
-    ip route show cache
-    ip route flush cache
-  fi
-  sleep 60
+	NOW=$(date "+%Y-%m-%d %H:%M:%S")
+	DROPPED_PACKETS=$(ovs-ofctl -O OpenFlow13 dump-flows br0 | sed -ne '/table=10,.* actions=drop/ { s/.* n_packets=//; s/,.*//; p }')
+	if [ "$DROPPED_PACKETS" != "" ] && [ "$DROPPED_PACKETS" -gt 1000 ];
+	then
+		echo "$NOW table=10 actions=drop packets=$DROPPED_PACKETS broken=true"
+	else
+		echo "$NOW table=10 actions=drop packets=$DROPPED_PACKETS broken=false"
+	fi
+	sleep 60
 done`
+	shellScriptDrop = `set -xe
+if [[ -f "/env/_master" ]]; then
+	set -o allexport
+	source "/env/_master"
+	set +o allexport
+fi
+
+echo "I$(date "+%m%d %H:%M:%S.%N") - drop-icmp - start drop-icmp ${K8S_NODE}"
+iptables -X CHECK_ICMP_SOURCE || true
+iptables -N CHECK_ICMP_SOURCE || true
+iptables -F CHECK_ICMP_SOURCE
+iptables -D INPUT -p icmp --icmp-type fragmentation-needed -j CHECK_ICMP_SOURCE || true
+iptables -I INPUT -p icmp --icmp-type fragmentation-needed -j CHECK_ICMP_SOURCE
+iptables -N ICMP_ACTION || true
+iptables -F ICMP_ACTION
+iptables -A ICMP_ACTION -j LOG
+iptables -A ICMP_ACTION -j DROP
+oc observe nodes -a '{ .status.addresses[1].address }' -- /tmp/add_iptables.sh
+tail -F /dev/null`
```

Comment:
is the tail intended like "sleep forever" ?  If so, why?  if the oc observe exits, why would we *not* want to retry?

Reply:
Right, we don't want this to block a restart.  Will update.